addons/l10n_it_edi/tests/test_withholding.py

```diff
@@ -75,7 +73,12 @@ def find_tax_by_ref(ref_name):
             'tax_ids': [(6, 0, [cls.inps_tax.id, cls.zero_tax.id])]
         }
 
-        invoice_data = cls.get_real_client_invoice_data()
+        cls.invoice_lines = [
```

Comment:
Why the change?

Reply:
The method was using `namedtuple` from `collections`, which is now flagged by `CI/style` (see [here](https://docs.astral.sh/ruff/rules/collections-named-tuple/)).
It seemed a bit overkill to create a dedicated class for that, plus it felt like the method was a bit useless a always returning the same values, so I thought it'd be easier to just store the values directly :nerd_face: